VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -29,18 +29,34 @@ import Foundation
 class VideoSettings: NSObject
 {
     var title: String?
+    {
+        didSet
+        {
+            self.title = trim(self.title)
+        }
+    }
+    
     var desc: String?
+    {
+        didSet
+        {
+            self.desc = trim(self.desc)
+        }
+    }
+    
     var privacy: String?
     var users: [String]? // List of uris of users who can view this video
     var password: String?
 
     init(title: String?, description: String?, privacy: String?, users: [String]?, password: String?)
     {
-        self.title = title
-        self.desc = description
+        super.init()
+        
         self.privacy = privacy
         self.users = users
         self.password = password
+        self.title = self.trim(title)
+        self.desc = self.trim(description)
```

Comment:
@alfiehanssen Hey sorry for taking so long to get back on this.  Unfortunately didSet is never called from init in swift.  

from the apple docs: 

> The willSet and didSet observers of superclass properties are called when a property is set in a subclass initializer, after the superclass initializer has been called. They are not called while a class is setting its own properties, before the superclass initializer has been called.

apparently wrapping it in a closure calls didSet.  Going to submit another pull request with that and the other updates.